website/README.md

```diff
@@ -4,8 +4,9 @@ This website is built using Docusaurus 2, a modern static website generator.
 
 ### Installation
 
+Install docusaurus with yarn:
 ```
-$ yarn
+$ yarn add docusaurus-init
```

Comment:
Curious, those are generated instructions by Docusaurus.

@yangshun, any thoughts about this? should the generated readme be improved?

Reply:
my bad, maybe it works actually

Reply:
No this is incorrect. The website has already been initialized and there's no need to add/run init again.  The existing instructions work fine.